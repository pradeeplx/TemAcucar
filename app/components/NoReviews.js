import React from 'react'
import { fontFactor } from "../helpers"
import Colors from "../Colors"
import Sentence from "./Sentence"

export default NoReviews = ({ user, currentUser }) => (
  <Sentence style={{
    alignSelf: 'stretch',
    margin: 10,
    textAlign: 'center',
    fontSize: 10 * fontFactor(),
  }}>
    {user.id === currentUser.id ? 'Você' : user.first_name} ainda não recebeu nenhuma avaliação.
  </Sentence>
)
