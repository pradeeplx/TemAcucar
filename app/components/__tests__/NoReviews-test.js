import 'react-native'
import React from 'react'
import NoReviews from '../NoReviews'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <NoReviews
      user={{ first_name: 'Foo', id: 1 }}
      currentUser={{ id: 1 }}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly when different user', () => {
  const tree = renderer.create(
    <NoReviews
      user={{ first_name: 'Foo', id: 1 }}
      currentUser={{ id: 2 }}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
