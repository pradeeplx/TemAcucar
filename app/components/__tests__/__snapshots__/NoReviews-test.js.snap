exports[`test renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#654C50",
        "fontFamily": "Avenir",
        "fontSize": 18,
      },
      Object {
        "alignSelf": "stretch",
        "fontSize": 15,
        "margin": 10,
        "textAlign": "center",
      },
    ]
  }>
  Você
   ainda não recebeu nenhuma avaliação.
</Text>
`;

exports[`test renders correctly when different user 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#654C50",
        "fontFamily": "Avenir",
        "fontSize": 18,
      },
      Object {
        "alignSelf": "stretch",
        "fontSize": 15,
        "margin": 10,
        "textAlign": "center",
      },
    ]
  }>
  Foo
   ainda não recebeu nenhuma avaliação.
</Text>
`;
